docker/conf/nginx.conf

```diff
@@ -48,8 +48,11 @@ http {
   server{
     listen              *:80;
     access_log          off;
+
+		set $meteor_proxy_pass_url http://127.0.0.1:3000;
```

Comment:
Indentation is off

Reply:
looks fine in vscode, not sure what github is doing to my commit :( 

Going to correct online rather than in my branch since it's showing correct there.